run_config_input.py

```diff
@@ -185,6 +186,34 @@ def run_normal_config_file(config_file="./examples/configuration.json"):
         chosen_dataset_option=chosen_dataset_option,
         distributed_data_parallelism=run_in_parallel,
     )
+
+    if "denormalize_output" in config and config["denormalize_output"] == "True":
+        dataset_path = f"{os.environ['SERIALIZED_DATA_PATH']}/serialized_dataset/{config['dataset_option']}.pkl"
+        with open(dataset_path, "rb") as f:
+            x_minmax = pickle.load(f)
+            y_minmax = pickle.load(f)
+        config["x_minmax"] = []
+        config["y_minmax"] = []
+        config["x_minmax"].append(x_minmax[:, :, config["atom_features"]].tolist())
+        if config["predicted_value_option"] == 1:
+            config["y_minmax"].append(y_minmax[:, 0].tolist())
+        elif config["predicted_value_option"] == 2:
+            config["y_minmax"].append(x_minmax[:, :, 1].tolist())
+        elif config["predicted_value_option"] == 3:
+            config["y_minmax"].append(x_minmax[:, :, 2].tolist())
+        elif config["predicted_value_option"] == 4:
+            config["y_minmax"].append(y_minmax[:, 0].tolist())
+            config["y_minmax"].append(x_minmax[:, :, 1].tolist())
+        elif config["predicted_value_option"] == 5:
+            config["y_minmax"].append(y_minmax[:, 0].tolist())
+            config["y_minmax"].append(x_minmax[:, :, 2].tolist())
+        elif config["predicted_value_option"] == 6:
+            config["y_minmax"].append(y_minmax[:, 0].tolist())
+            config["y_minmax"].append(x_minmax[:, :, 1].tolist())
+            config["y_minmax"].append(x_minmax[:, :, 2].tolist())
+    else:
+        config["denormalize_output"] = "False"
```

Comment:
I like the idea of setting defaults like this to not break compatibility with existing configurations